<!-- // 2024 Rebuttal -->
<launch>
    <node name="data" pkg="alan_visualization" type="vicon_tracking" output="screen">
        <param name="/log_path" type="string" value="/home/patty/alan_ws/src/alan/alan_visualization/log/2024rebuttal/"/>
        <param name="/filename1" type="string" value="rebuttal_land_tracking.csv"/>
        <!-- <param name="/filename2" type="string" value="0719_land_uav.csv"/> -->
    </node>         
</launch>
